src/Markup/Avalonia.Markup.Xaml.Loader/CompilerDynamicDependencies.cs

```diff
@@ -0,0 +1,133 @@
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Metadata.DependsOnAttribute))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Metadata.IAddChild))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Metadata.IAddChild<>))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Metadata.InheritDataTypeFromItemsAttribute))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Metadata.MarkupExtensionDefaultOptionAttribute))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Metadata.MarkupExtensionOptionAttribute))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Metadata.TemplateContentAttribute))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Metadata.TrimSurroundingWhitespaceAttribute))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Metadata.UsableDuringInitializationAttribute))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Metadata.WhitespaceSignificantCollectionAttribute))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Metadata.XmlnsDefinitionAttribute))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Point))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.RelativePoint))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.RelativeUnit))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.StyledElement))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.StyledElementExtensions))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.StyledProperty<>))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Styling.ControlTheme))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Styling.IStyle))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Styling.Selectors))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Styling.SetterBase))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Styling.Style))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Styling.ThemeVariant))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.UnsetValueType))]
+    [DynamicDependency(XamlAccessible, typeof(Avalonia.Utilities.TypeUtilities))]
+    [DynamicDependency(XamlAccessible, typeof(System.ComponentModel.CultureInfoConverter))]
+    [DynamicDependency(XamlAccessible, typeof(System.ComponentModel.ISupportInitialize))]
+    [DynamicDependency(XamlAccessible, typeof(System.ComponentModel.ITypeDescriptorContext))]
+    [DynamicDependency(XamlAccessible, typeof(System.ComponentModel.TypeConverterAttribute))]
+    [DynamicDependency(XamlAccessible, typeof(System.Globalization.CultureInfo))]
+    [DynamicDependency(XamlAccessible, typeof(System.IFormatProvider))]
+    [DynamicDependency(XamlAccessible, typeof(System.IServiceProvider))]
+    [DynamicDependency(XamlAccessible, typeof(System.Uri))]
+    [DynamicDependency(XamlAccessible, typeof(System.UriKind))]
+    [DynamicDependency(XamlAccessible, typeof(System.Windows.Input.ICommand))]
+    public CompilerDynamicDependenciesAttribute()
```

Comment:
`System.Collections.Generic.IReadOnlyList<>` from `AvaloniaXamlIlWellKnownTypes` seems missing (it's very likely kept by another type but let's be sure).

I assume that generating this list wasn't worth the amount of work since it's unlikely to change often?

Reply:
Actually, I wasn't really thinking about source-gen solution. It also shouldn't be very difficult to implement. Let me try.

Reply:
Done